### PR TITLE
#67: Fixed Python version for AWS Codebuild 

### LINKED ISSUE #67: Wrong Python version for AWS codebuild
Python Version in AWS codebuild also needs to be bumped to 3.10

### PR DESCRIPTION
**Pull request by tomuben**
fixes #67

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -30,6 +30,7 @@ toml = ">=0.10.2"
 pytest = "^7.1.1"
 pytest-mock = "^3.7.0"
 cfn-lint = "^0.75.1"
+exasol-toolbox = "^0.12.0"
```
Comment (tkilias) [inline pyproject.toml:33]: for what are you using the toolbox?

**Review by tkilias — DISMISSED** [review]

File: pyproject.toml (hunk 1 above)
Comment (ahsimb) [inline pyproject.toml:33]:
```suggestion
exasol-toolbox = ">=0.12.0,<1.0.0"
```

Comment (ahsimb) [reply pyproject.toml:33]: When the version is less than 1.0.0 the `^` increments the second number, not the first to get the upper boundary.

**Review by ahsimb — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (tomuben) [reply pyproject.toml]: Good point

Comment (tomuben) [reply pyproject.toml]: But I removed toolbox dependency for now

Comment (tomuben) [reply pyproject.toml]: I wanted to use it for automatic release PR creation (which adapts the changelog and versions). But I didn't manage to use it....
So, I removed the dependency again.

**Pull request merged**